client/src/pages/AddTrainee.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button } from "../component/Button";
+import "../pages/addTrainee.css";
+export const AddTrainee = () => {
+	const [fullname, setFullName] = useState("");
+	const [githubUserName, setGithubUserName] = useState("");
+    const navigate = useNavigate();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			const response = await fetch("/api/cohorts/{cohortId}/trainees", {
+				method: "POST",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify({
+					full_name: fullname,
+					github_user_name: githubUserName,
+				}),
+			});
+
+			if (!response.ok) {
+				throw new Error("couldn't to add trainee");
+			}
+
+			console.log("Trainee added successfully");
```

Comment:
No need to log this on the browser as it wont go anywhere, useful for debugging but should be remove for production

Reply:
I just to check it's working and forgot to delete it, Fixed